database/migrations/2022_07_13_174800_permission_test.php

```diff
@@ -12,7 +13,14 @@ class PermissionTest extends Migration
 	 */
 	public function up(): void
 	{
+		$is_public = Str::endsWith(getcwd(), 'public');
+		if ($is_public) {
+			chdir('..');
+		}
 		Artisan::call('lychee:fix-permissions', ['--dry-run' => 1]);
+		if ($is_public) {
+			chdir('public');
+		}
 	}
```

Comment:
This might be effective but it looks just nasty! Wouldn't it be better to fix `app/Console/Commands/FixPermissions.php` to use absolute pathnames instead (via `Storage::disk('images')` and `Storage::disk('symbolic')`, I guess)?

Reply:
Fixed with [adf57ba](https://github.com/LycheeOrg/Lychee/pull/1495/commits/adf57badf1e7e205591c09db2c5cae2e3a61ede7)